forest/src/daemon.rs

```diff
@@ -102,6 +103,12 @@ pub(super) async fn start(config: Config) {
         ks.put(JWT_IDENTIFIER.to_owned(), generate_priv_key())
             .unwrap();
     }
+
+    // Print admin token
+    let ki = ks.get(JWT_IDENTIFIER).unwrap();
+    let token = create_token(ADMIN.to_owned(), ki.private_key()).unwrap();
+    println!("Admin token: {}", token);
```

Comment:
Not understanding why we need to create a new JWT upon launch.

Reply:
It's not a new key, it's just a token derived from the existing key. Because I've added rich permissioning for every endpoint, there's no way to call `auth create-token` without providing an admin JWT. If you're using an encrypted keystore, this then becomes impossible to use.